parcelgen.py

```diff
@@ -199,7 +208,7 @@ def needs_jsonutil(self):
         if "Date" in self.props:
             return True
         for key in self.props.keys():
-            if "List" in key:
+            if "List" or "Map" in key:
```

Comment:
haha. weird I have tested it but now I realised I haven't tested it with a False response, so yep, it doesn't work indeed.